tests/lit-tests/loop_unroll_varying.ispc

```diff
@@ -43,21 +43,13 @@ void foo_for() {
 // CHECKO0:         br label %{{[a-zA-Z_][a-zA-Z0-9_.]*}}
 // CHECKO0:         }
 
-// CHECKO2:         for_loop{{.*}}:
 // CHECKO2:         call void @goo_for_unroll8___
```

Comment:
If this test count the number of calls to `goo_for_unroll8___`, so it probably needs to check that with `CHECK-COUNT-N`.

Reply:
Fixed.